index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="fill-100">
+    <nav class="navigation">
+      <span>Logo</span>
+      <span>Menu</span>
+    </nav>
+  </header>
+  <main class="fill-100">
+    <article>
+      <section>
+        <p>I'm Javi,</p>
+        <h1> A Web Designer that can code</h1>
+      </section>
+      <section>
+        <p>I make ads and websites, combined with code to improve the conversion rates of their marketing</p>
+      </section>
+      <section class="pv">
+        <a href="http://">Let's connect</a>
+      </section>
+      <section>
+        <span>Link</span>
+        <span>Link</span>
+        <span>Link</span>
+      </section>
```

Comment:
- [ ] Please kindly rearrange your lines of code, you only need just one section element to wrap all other elements. ℹ️